LunaSysMgr.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Luna System Manager
-After=LunaSysService.service nyx.target
+After=LunaSysService.service nyx.target configurator.service
 Requires=LunaSysService.service
```

Comment:
I don't have write access here, so can't merge myself :) But just having After is correct, i think. See my other reply for mediaindexer PR.